tests/test_ilastik/test_applets/dataSelection/testOpDataSelection.py

```diff
@@ -135,7 +135,7 @@ def setup_class(cls):
                 )
 
         # Create a 'project' file and give it some data
-        cls.projectFile = h5py.File(cls.projectFileName)
+        cls.projectFile = h5py.File(cls.projectFileName, "a")
```

Comment:
Minor nitpick, but does this test project file really need the `a` mode, or `w` should suffice? Ditto for the other uses of `a`.

Reply:
thx @emilmelnikov, exactly the right amount of rigorous attention!